operator/v1/types_openshiftcontrollermanager.go

```diff
@@ -0,0 +1,44 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// OpenShiftControllerManagerConfig provides information to configure openshift-controller-manager
+type OpenShiftControllerManagerConfig struct {
```

Comment:
is this used or cruft?  I didn't end up using it in most of them.

Reply:
looks like it is not, will remove.